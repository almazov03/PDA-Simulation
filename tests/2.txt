STARTING_NON_TERMINAL=$S$
$S$ = {$X$ $A$}
$S$ = {$B$ $B$}
$B$ = {"b"}
$B$ = { $S$ $B$ }
$X$ = { "b" }
$A$ = { "a" }