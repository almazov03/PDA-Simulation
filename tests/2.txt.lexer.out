LexToken(START,'S',1,0)
LexToken(NON_TERMINAL,'S',2,27)
LexToken(EQUAL,'=',2,31)
LexToken(LBR,'{',2,33)
LexToken(NON_TERMINAL,'X',2,34)
LexToken(NON_TERMINAL,'A',2,38)
LexToken(RBR,'}',2,41)
LexToken(NON_TERMINAL,'S',3,44)
LexToken(EQUAL,'=',3,48)
LexToken(LBR,'{',3,50)
LexToken(NON_TERMINAL,'B',3,51)
LexToken(NON_TERMINAL,'B',3,55)
LexToken(RBR,'}',3,58)
LexToken(NON_TERMINAL,'B',4,61)
LexToken(EQUAL,'=',4,65)
LexToken(LBR,'{',4,67)
LexToken(TERMINAL,'b',4,68)
LexToken(RBR,'}',4,71)
LexToken(NON_TERMINAL,'B',5,74)
LexToken(EQUAL,'=',5,78)
LexToken(LBR,'{',5,80)
LexToken(NON_TERMINAL,'S',5,82)
LexToken(NON_TERMINAL,'B',5,86)
LexToken(RBR,'}',5,90)
LexToken(NON_TERMINAL,'X',6,93)
LexToken(EQUAL,'=',6,97)
LexToken(LBR,'{',6,99)
LexToken(TERMINAL,'b',6,101)
LexToken(RBR,'}',6,105)
LexToken(NON_TERMINAL,'A',7,108)
LexToken(EQUAL,'=',7,112)
LexToken(LBR,'{',7,114)
LexToken(TERMINAL,'a',7,116)
LexToken(RBR,'}',7,120)
